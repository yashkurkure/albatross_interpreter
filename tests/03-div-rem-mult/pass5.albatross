printint(38+101|175/115&47^251-132-8%10|195/101%81&41+96|2^123%57/23&203/132-97%42%58^58);
printint(149&251|103/186&45/124%97%139-8/221*214&120-216^221-246/213%142);
printint(23/145*204&50/181/76%194^83*156-158%93/121|165/109-77-160);
printint(245+65^181%97&9-156%136^95*149|132^239);
printint(63|58+197%32-125|241^99|154+164*168&72-250^87%97+173^172*160+254^129&174^120%44&233);
printint(87^169%124/121|155*143/140^86+5%78%42^32%56-220^191^182-230-136&159&37+233+158-246*235^161|253%62);
printint(120+176&36&250/169^111*1^191%222^194%87-42%35^49-86-107%10+207*99);
printint(151/135%143-140^185&47^18^40-81*64-214-117&175%242-24*188*21%1%30-77%217-236&101+133^117);
printint(167/215/84|164|48+217^56/71-30&144-142+115|158+237);
printint(87|67+58/221-155|32+225%36%201^208|7*133%18-208/152%255*95/243&231-115|245&224|121);
return 0;
