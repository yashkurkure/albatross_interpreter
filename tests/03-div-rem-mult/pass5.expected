255221-10813931-1-4120284-197495-1
